src/app/routes/registration/request-claim/request-claim.component.ts

```diff
@@ -276,9 +281,10 @@ export class RequestClaimComponent implements OnInit, SubjectElements {
         registrationTypes: enrolForm.registrationTypes,
       } as any);
 
+      // TODO: remove any, add proper typing to the form
       this.displayAlert(
         'Request to enrol as ' +
-          this.roleTypeForm.value.roleType.name.toUpperCase() +
+          (this.roleTypeForm.value.roleType as any).name.toUpperCase() +
```

Comment:
Would the way to fix this be to have `roleType` in the `IRoleTypeForm` interface be `{ name: string }`?